### DISCUSSION TITLE
Where can I find the GPU driver?

### DISCUSSION THREAD

**Discussion by PoTaTo625** [skyline-emu/skyline#1801] [Q&A] [answered]
I don't know where to find them.

**Comment by Crytonics** [comment, marked as answer]
We have discord server. You can find drivers in "Drivers" channel.
https://discord.gg/XnbXNQM

**Reply by PoTaTo625** [reply]
Unfortunately, I'm in China and I can't get the verification code. Thank you very much!😃

**Reply by tmzy00** [reply]
可以的，你用电话号码注册选择+86

**Reply by PoTaTo625** [reply]
我有账号，后来登陆的时候他说我登陆异常，要验证手机，可是收不到验证码:(

**Reply by tmzy00** [reply]
你可来这个QQ群633327980，里面我分享了

**Reply by PoTaTo625** [reply]
好！谢谢
